### DISCUSSION TITLE
How to run the Hybrid app?

### DISCUSSION THREAD

**Discussion by Take-A-Byte** [dotnet/eShop#97] [General]
It does not show in .slnf solution with Preview version of Visual Studio. And when I use .sln file, there's an error when running the project.

Could someone please point me to the set of instructions to run this app.

**Comment by jamesmontemagno** [comment]
@Eilon

**Comment by Eilon** [comment]
Hi @Take-A-Byte if you have Visual Studio 17.8 with the .NET MAUI workload installed (along with other pre-reqs) then opening the SLN should get you everything you need.

To run the .NET MAUI Hybrid app you'll need to:

1. Run the **eShop.AppHost** project to get all the web APIs up and running using Aspire and leave it running. An easy way to do this is to right-click on the project in Solution Explorer, select **Debug**, and then **Run without Debugging**.
2. While that's still running, run the **HybridApp** project in the same way

And that should be all you need.

If you're seeing any errors, please share details/logs/screenshots of what you're seeing and we'll try to figure it out. Thanks!

**Comment by adanbrownpaca** [comment]
After following the steps outline above, I got:

![Screenshot 2024-01-05 151526](https://github.com/dotnet/eShop/assets/135455142/25997c6f-135e-4372-a9a7-fa395fa154ab)
